packages/ui-components/src/components/inline-editable-field/test/inline-editable-field.e2e.ts

```diff
@@ -33,7 +33,7 @@ describe('KvInlineEditableField (e2e tests)', () => {
 			expect(btns.length).toBe(0);
 		});
 
-		describe('and focus the component', () => {
+		describe.skip('and focus the component', () => {
```

Comment:
we can't catch the focused element?

Reply:
This is a leftover, I will revert this change.